cypress/integration/pages/summary.spec.js

```diff
@@ -12,7 +12,14 @@ context('Summary page', () => {
         targetReviewDate: '2022-05-29T00:00:00.000Z',
         text: 'some text',
         useAsPhp: false,
-        agreedWithName: 'Ami Working'
+        agreedWithName: 'Ami Working',
+        actions: [
+          {
+            summary: 'text',
+            description: 'http://localhost:3000',
+            dueDate: '2020-08-08T00:00:00.000Z'
```

Comment:
`dueDate` is stored as only the date like `2020-08-08`. This will still work, but we should probably match how it actually gets stored. There is also an `isCompleted` value that we need here.

(We should refactor these tests in future to use a function or something to generate test data, that way we can avoid having to update all our tests when we change things!)

Reply:
Ah, good one!